guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -80,11 +82,18 @@ If you need to change the Fastly API key credential for a project environment fo
 
 ### Multiple Fastly accounts and assigned domains {#domain}
 
-If you have an existing Fastly account that links the same apex and sub-domains used for your {{ site.data.var.ece }} project, remove those domains from your Fastly account before submitting the request to add Fastly services to your project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation. 
+Fastly only allows you to assign an apex domain and associated subdomains to one Fastly service and account. If you have an existing Fastly account that links the same apex and subdomains used for your {{ site.data.var.ece }}, remove those domains from the existing account before submitting the request to add Fastly services to your project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation.
+
+If you have other sites hosted on the same apex subdomain and want to continue managing Fastly services from the existing account, you can leave the apex domain linked to that account and transfer the subdomains for your project to the {{ site.data.var.ece }} Fastly service account. To use this type of configuration, you must submit a Fastly support ticket requesting delegation of the apex domain.
+
+For example, for a project with the following conditions and requirements, you would submit a [Fastly support ticket](https://fastly.zendesk.com/hc/en-us/articles/204950779-Filing-support-tickets) to delegate `testweb.com` to Magento, including the {{ site.data.var.ece }} project ID in the ticket:
 
+-  The {{ site.data.var.ece }} project uses the `storeprod.testweb.com` and `storestaging.testweb.com` subdomains
```

Comment:
These bullet items don't use a consistent pattern

Reply:
Mainly for review-- total revision based on technical review